lighthouse-core/lib/network-recorder.js

```diff
@@ -368,7 +368,7 @@ class NetworkRecorder extends EventEmitter {
     for (const record of records) {
       const stackFrames = (record.initiator.stack && record.initiator.stack.callFrames) || [];
       const initiatorURL = record.initiator.url || (stackFrames[0] && stackFrames[0].url);
-      const initiator = recordsByURL.get(initiatorURL) || record.redirectSource;
```

Comment:
presumably there was some reason I fellback this way but I have no idea what is was at this point and this seems to make more sense, so..... cool!

maybe add a comment so we don't forget again?